### DISCUSSION TITLE
An official Android app release on Google play and on F-Droid.

### DISCUSSION THREAD

**Discussion by NotYeMK** [monkeytypegame/monkeytype#2178] [Ideas]
As someone who travels a lot and doesn't have a constant internet connection, I would really love to see an Android app version of this so it can be used offline and on the go. I want to keep on practicing typing faster while I am away from internet for long periods of time. This would really help out with that and make my travels so much better.

**Comment by tajhans** [comment]
This has been answered in this issue: #1018.

**Reply by NotYeMK** [reply]
Not a PWA form because I don't fw chromium browsers and just don't like webapps in general. So no, this didn't add any form of valuable discussion because I was asking for a standalone app and not a PWA.
